static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx

```diff
@@ -15,10 +24,10 @@ type Span = (RawSpanType | TraceContextSpanProxy) & {
 
 type SpanEvidenceKeyValueListProps = {
   causeSpans: Span[] | null;
+  event: EventTransaction;
   issueType: IssueType | undefined;
   offendingSpans: Span[];
   parentSpan: Span | null;
-  transactionName: string;
 };
```

Comment:
I agree that's better, but I don't want this PR to balloon! Is it too confusing to keep it as-is?

Reply:
I may be biased because I wrote the span evidence component but I don't think it will make the PR harder to read 😄 

It would be nicer if we could add that refactor to this PR as well

Reply:
Alright, as you wish, but now this PR is much more about the refactor than the actual feature!

Reply:
I suppose I underestimated the amount of refactoring required lol

In this case we should rename the title of this PR to include this added scope. Currently taking a look at the changes